roles/matrix-bridge-appservice-discord/templates/config.yaml.j2

```diff
@@ -58,8 +58,11 @@ database:
   # If you are migrating, see https://github.com/Half-Shot/matrix-appservice-discord/blob/master/docs/howto.md#migrate-to-postgres-from-sqlite
   # WARNING: You will almost certainly be fine with sqlite unless your bridge
   # is in heavy demand and you suffer from IO slowness.
-  filename: "/data/discord.db"
-  # connString: "postgresql://user:password@localhost/database_name"
+  {% if matrix_appservice_discord_database_engine == 'sqlite' %}
+  filename: {{ matrix_appservice_discord_database_filename|to_json }}
+  {% else %}
+  connString: {{ matrix_appservice_discord_database_connString|to_json }}
```

Comment:
`|to_json`? That should be a string already, and if it weren't, there's `|string`..

Reply:
It should be a string already, but if it contains some special characters, it has the chance of breaking YAML parsing.

Using `|to_json` ensures that it would do proper double-quote wrapping and escape any other weird characters that may be inside, so that it would be valid YAML syntax.

It's probably unlikely and mostly affecting `matrix_appservice_discord_database_connString` and not `matrix_appservice_discord_database_filename`, but it doesn't hurt having it there.

I think our default connection strings yield rather safe values, but people might set a weird password or something sometimes.

We've recently had complains of ma1sd breaking, because someone's Postgres password contained a weird sequence (e.g. `:{space}`). That was fixed here: https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/7372480e95d9fa60c05270236cc6f4d6753c3e67